### PR TITLE
1 Create framework

### PR DESCRIPTION
**Pull request by antonKoiko**

### CODE REVIEW

File: framework.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import sys
+import time
+import threading
+
+def signal_handler(sig, frame):
+    print('\nYou pressed Ctrl+C or sent a SIGTERM signal. Shutting down gracefully...')
+    sys.exit(0)
+
+def print_time():
+
```
Comment (eqld) [inline framework.py]: Remove unnecessary new line

File: framework.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+def main():
+    signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
+    print('Press Ctrl+C or send a SIGTERM signal to stop')
+    t = threading.Thread(target=print_time)
+    t.start()
+
+if __name__ == "__main__":
+    main()
```
Comment (eqld) [inline framework.py]: Missing new line after `main()`

**Review by eqld — CHANGES_REQUESTED** [review]

File: framework.py (hunk 1 above)
Comment (antonKoiko) [reply framework.py]: done

File: framework.py (hunk 2 above)
Comment (antonKoiko) [reply framework.py]: done

**Review by eqld — APPROVED** [review]
lgtm

**Pull request merged**